src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+    private static final int ARRAY_SIZE = 6;
+    private static final double MAX_SIZE = 10.0;
+    private static final Random RANDOM = new Random();
```

Comment:
random shouldn't be constant. Let's make it lowercased and remove static keywork. And also place your colorSupplier here